app/assets/stylesheets/profile.scss

```diff
@@ -0,0 +1,83 @@
+// Edit Page
+#profile-picture {
+  min-height: 400px;
+}
+
+#update-profile-title {
+  margin-top: 1em;
+  font-weight: 300;
+}
+
+#form-container {
+  padding: 3em 1em;
+  font-family: $noto-font;
+}
+
+.heavy-labels label {
```

Comment:
looks like this is only being used once, maybe just add styling under the id form-container. Not a blocker though! Just mentioning it because this file is rather long !